server/src/grpc/remote_engine_service/mod.rs

```diff
@@ -630,7 +690,7 @@ impl RemoteEngineServiceImpl {
             // The first request, need to handle it, and then notify the other requests.
             RequestResult::First => {
                 let query = async move {
-                    handle_execute_plan(ctx, encoded_plan, query_engine)
+                    handle_execute_plan(ctx, physical_plan_clone, query_engine)
                         .await
                         .map(PartitionedStreams::one_stream)
                 };
```

Comment:
Just need to modify the return value of `handle_execute_plan`, wrapp it as saying above.